app/component/StopCardHeader.js

```diff
@@ -28,8 +29,34 @@ class StopCardHeader extends React.Component {
     return description;
   }
 
+  getExternalLink(code, isPopUp) {
+    // Check for popup from stopMarkerPopup, should the external link be visible
+    if (!code || isPopUp) {
```

Comment:
this.headerConfig.virtualMonitorBaseUrl check should probably be moved here as well and the later one can be removed-